CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.0
+_Not released yet_
+
+- Breaking Change: Athena plugin now requires Grafana 8.0+ to run.
```

Comment:
I am not sure if this _needs_ to be a breaking change. I believe that I tried to run a plugin with an unsupported Grafana version and it worked. It may be worth to test that.

Reply:
were you able to check this?

Reply:
You're right, it looks like the `grafanaDependency` is not blocking you from running a plugin on an unsupported version. 
In the case of Athena there are no further changes (all the deps in `package.json` were already 8.0+) so non-breaking. 

I'll update the changelog

Reply:
🤔 
 Would you say stopping support for older version of grafana (with no breaking changes) warrants a Major, Minor or Patch increment?

Reply:
how did you check this blocking behaviour, is it just locally with the Grafana plugins folder? I believe the `grafanaDependency` doesn't stop you from using the plugin that way (which it probably should, or at least warn you about incompatibility) but is more for the Grafana plugin site for the `Dependencies` field e.g. https://grafana.com/grafana/plugins/grafana-athena-datasource/

I would have also thought you wouldn't be able to install plugins via both the site and Grafana instance (e.g. non-locally and with both on-prem and Cloud) with unsupported versions but I haven't checked this behaviour lately. I'm also unsure about the behaviour for AMG too since these are always installed.

Good question re the major/minor/patch for the minimum required Grafana version. I would have thought you would match the version change of the required version e.g. 7 -> 8 is major so this would also be a major change or 8.1 -> 8.2 is minor so this would be also be minor. Might be worth discussing further with the team because we haven't talked about this before though.

Reply:
Make sense, I'll keep `2.0.0` then